### PR TITLE
docs: fix ambiguity.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Looks better indeed :+1:

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ def query(filename):
 data = query("sample1.flac")
 ```
 
-You can also use libraries such as [transformers](https://huggingface.co/models?library=transformers&pipeline_tag=automatic-speech-recognition&sort=downloads), [speechbrain](https://huggingface.co/models?library=speechbrain&pipeline_tag=automatic-speech-recognition&sort=downloads), [NeMo](https://huggingface.co/models?pipeline_tag=automatic-speech-recognition&library=nemo&sort=downloads) and [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=automatic-speech-recognition&sort=downloads) if you want one-click managed Inference without any hassle.
+We also support libraries such as [transformers](https://huggingface.co/models?library=transformers&pipeline_tag=automatic-speech-recognition&sort=downloads), [speechbrain](https://huggingface.co/models?library=speechbrain&pipeline_tag=automatic-speech-recognition&sort=downloads), [NeMo](https://huggingface.co/models?pipeline_tag=automatic-speech-recognition&library=nemo&sort=downloads) and [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=automatic-speech-recognition&sort=downloads) via the Serverless Inference API.
```
Comment (osanseviero) [inline packages/tasks/src/tasks/automatic-speech-recognition/about.md:39]: The flow is still a bit weird to me. THis is how I read it:

1. There are 17k models, and you can use them using the Inference API
2. Code snippet using API (python)
3. You can use multiple libraries with the api
4. Code snippet using `transformers`
5. Code snippet using `API (javascript)

4 feels really out of place here.

Comment (Vaibhavs10) [reply packages/tasks/src/tasks/automatic-speech-recognition/about.md]: Updated in the latest commit! Thoughts?

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md [hunk 2]
```diff
@@ -36,7 +36,19 @@ def query(filename):
 data = query("sample1.flac")
 ```
 
-You can also use libraries such as [transformers](https://huggingface.co/models?library=transformers&pipeline_tag=automatic-speech-recognition&sort=downloads), [speechbrain](https://huggingface.co/models?library=speechbrain&pipeline_tag=automatic-speech-recognition&sort=downloads), [NeMo](https://huggingface.co/models?pipeline_tag=automatic-speech-recognition&library=nemo&sort=downloads) and [espnet](https://huggingface.co/models?library=espnet&pipeline_tag=automatic-speech-recognition&sort=downloads) if you want one-click managed Inference without any hassle.
+You can also use [huggingface.js](https://github.com/huggingface/huggingface.js) to transcribe text with javascript using models available on the Hugging Face Hub.
```
Comment (osanseviero) [inline packages/tasks/src/tasks/automatic-speech-recognition/about.md:39]:
```suggestion
You can also use[huggingface.js](https://github.com/huggingface/huggingface.js), the JavaScript client, to transcribe models with the Inference API.
```

**Review by osanseviero — APPROVED** [review]
Thanks, this is much clearer! :+1:

File: packages/tasks/src/tasks/automatic-speech-recognition/about.md (hunk 1 above)
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/automatic-speech-recognition/about.md]: Noice! Resolving this as fixed.

**Pull request merged**